client-app/core/composables/useLanguages.ts

```diff
@@ -10,134 +10,143 @@ import type { I18n } from "@/i18n";
 import type { LocaleMessage } from "@intlify/core-base";
 import type { Composer } from "vue-i18n";
 
-const { themeContext } = useThemeContext();
+function _useLanguages() {
+  const { themeContext } = useThemeContext();
 
-const pinedLocale = useLocalStorage<string | null>("pinedLocale", null);
+  const pinedLocale = useLocalStorage<string | null>("pinedLocale", null);
```

Comment:
what the problem with this file? Only because of createGlobalState? Move it to refactoring with language fallback pull request